src/payee/handler.js

```diff
@@ -136,24 +138,30 @@ exports.getPartiesByTypeAndId = function (req, h) {
       const opts = {
         method: 'PUT',
         headers: {
-          Accept: 'application/vnd.interoperability.parties+json;version=1',
           'Content-Type': 'application/vnd.interoperability.parties+json;version=1.0',
           'FSPIOP-Source': 'payeefsp',
           'FSPIOP-Destination': req.headers['fspiop-source'],
           Date: new Date().toUTCString(),
           'FSPIOP-Signature': JSON.stringify(fspiopSignature),
           'FSPIOP-HTTP-Method': 'PUT',
           'FSPIOP-URI': `/parties/${req.params.type}/${req.params.id}`,
-          traceparent: req.headers.traceparent ? req.headers.traceparent : undefined,
-          tracestate: req.headers.tracestate ? req.headers.tracestate : undefined
+          traceparent: req.headers.traceparent ? req.headers.traceparent : '',
+          tracestate: req.headers.tracestate ? req.headers.tracestate : ''
         },
-        rejectUnauthorized: false,
-        body: JSON.stringify(myCache.get(req.params.id))
+        transformRequest: [(data, headers) => {
+          delete headers.common.Accept
+          return data
+        }],
+        httpsAgent: new https.Agent({
+          rejectUnauthorized: false
+        }),
+        data: JSON.stringify(myCache.get(req.params.id))
       }
+
       Logger.info((new Date().toISOString()), 'Executing PUT', url)
-      const res = await fetch(url, opts)
+      const res = await request(url, opts)
```

Comment:
I dont believe that this would work. axios requires the url to be sent in the opts. therefore your call should look like 
```
const opts = {
    url: '',
    method: 'PUT',
    headers: [],
    ....
}
const res = await request(opts)
```

Reply:
Did you actually try it out?